### DISCUSSION TITLE
Cura v5 support

### DISCUSSION THREAD

**Discussion by pierremilcent** [5axes/Calibration-Shapes#95] [General]
Any idea if and when you are going to support Cura v5 ? Thanks for your helpful contribution to the community with such amazing plugins.

**Comment by 5axes** [comment]
So the answer is yes , but not from the marketplace , until to have a stable release

**Reply by contagous** [reply]
They finally have a stable release for 5, what's the ETA do you think on the calibration support?

**Reply by 5axes** [reply]
ETA ?

**Reply by contagous** [reply]
Estimated Time Arrival. When is due to come out

**Reply by 5axes** [reply]
Ok I was not aware of this acronym. Well as you have seen a version is available on this github. Concerning the marketplace of Cura it's a decision of the Ultimaker developpers. They need to check the code, test it, then release it. I would say usualy between 10 days and 1 month. Depends if they are busy with a new release or not. But during this laps of time, if we find an issue or if  we made a modification in the plugin submission , we start for a new cycle of 10 days

**Reply by contagous** [reply]
Ok thanks for the update
